controllers/User.js

```diff
@@ -145,6 +145,8 @@ module.exports = {
       return ctx.badRequest(null, [{ messages: [{ id: 'No authorization header was found' }] }]);
     }
 
+    user = strapi.plugins['users-permissions'].services.user.fetch({ id: user.id }, ['artworks', 'collected', 'memoirs', 'interviews', 'orders', 'links'])
```

Comment:
This will throw an error, user L142 needs to be an `let` instead of `const`

Reply:
you have really good eyes! 
It is changed, thank you!